rero/rero-ils
# items: fix online locations status

* Fix `data/location.json` file : According to PO, removing location named 'Online' with no flag `is_online=True`
* Closes #573

Co-Authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## How to test?

- run `scripts/setup`
- run `curl -XPOST -H 'Content-Type: application/json' "http://localhost:9200/locations/_search?pretty" -d '{"query": {"term": { "is_online": true } } } }'` to get online location pids. Result must be [12,18,4]
- run `curl -XPOST -H 'Content-Type: application/json' "http://localhost:9200/items/_search?pretty" -d '{"query": {"terms": { "location.pid": [12, 18, 4] } } }'` to get number if items in these locations. The number should be "0"

## Code review check list
